indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -436,6 +437,28 @@ def can_review(self, user):
         else:
             return False
 
+    def is_convener(self, user):
+        if not user:
+            return False
+        elif not self.event_new.can_manage(user, role='track_convener'):
+            return False
+        elif self.event_new in user.global_convener_for_events:
+            return True
+        elif user.convener_for_tracks & self.reviewed_for_tracks:
+            return True
+        else:
+            return False
+
+    def can_judge(self, user):
+        if not user:
+            return False
+        elif self.event_new.can_manage(user):
+            return True
+        elif self.is_convener(user):
+            return abstracts_reviewing_settings.get(self.event_new, 'allow_convener_judgment')
```

Comment:
What about using the CFA property I had added in my PR? https://github.com/OmeGak/indico/commit/b0f52a762117a5c5278f5a2f81cb84aa8c2f64af